### PR TITLE
Proper Radio Channels for Borgs

### PR DESCRIPTION
**Pull request by Surrealaser**
## About The Pull Request

Borgs now gain access to role appropriate radio channels:

Medborgs get access to Medical.
Peacekeepers and Secborgs get access to Security.
Engiborgs get access to Engineering.
Mining borgs get access to Mining.

All other borgs get access to Service.

## Why It's Good For The Game

Helps borgs do their job by allowing them to be in better communication with their department and take appropriate cues.

## Changelog
:cl:
add: Borgs now gain access to role appropriate radio channels.
/:cl:

### CODE REVIEW

**Comment by Kokojos** [comment]
Quality of life ++.
Worth it to merge.

**Comment by TheSwain** [comment]
Okay gonna add this in, just needs a conflict resolve and we're good

File: code/zFulpstationCode/fulp_overwrite_vars.dm [hunk 1]
```diff
@@ -37,3 +37,63 @@
...
+
+/mob/living/silicon/robot/modules/medical
+	radio = /obj/item/radio/borg/medical
+
+/obj/item/radio/borg/medical
+	keyslot = new /obj/item/encryptionkey/headset_med
+
+
+/mob/living/silicon/robot/modules/engineering
+	radio = /obj/item/radio/borg/medical
```
Comment (remove32) [inline code/zFulpstationCode/fulp_overwrite_vars.dm]: typo

**Comment by remove32** [comment]
doesnt seem to be working 
image

**Review by remove32 — CHANGES_REQUESTED** [review]

**Comment by Surrealaser** [comment]
Conflicts and issues should be fixed; PR has been tested.

**Comment by remove32** [comment]
can you tick the new file in the dme

**Comment by Surrealaser** [comment]
@remove32 I mean they were ticked in the version I saved and uploaded

**Comment by Surrealaser** [comment]
https://i.imgur.com/9dqwJ4C.png

**Comment by Surrealaser** [comment]
@TheSwain @remove32 This should be ready for merge.

**Review by remove32 — APPROVED** [review]

**Pull request merged**